### PR TITLE
Add combined memory and swap meter

### PR DESCRIPTION
**Pull request by cgzones**
Closes: #699

### CODE REVIEW

File: MemorySwapMeter.c [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+   this->mode = mode;
+
+   Meter_setMode(data->memoryMeter, mode);
+   Meter_setMode(data->swapMeter, mode);
+
+   this->h = MAXIMUM(Meter_modes[data->memoryMeter->mode]->h, Meter_modes[data->swapMeter->mode]->h);
+}
+
+
```
Comment (BenBE) [inline MemorySwapMeter.c:80]: Can drop this second blank line …

**Review by BenBE — APPROVED** [review]

**Review by natoscott — APPROVED** [review]
LGTM.

**Comment by BenBE** [comment]
@cgzones Which part of this is still WIP?

**Comment by cgzones** [comment]
In text-mode the memory meter text is truncated (maybe it should switch between a wide and a compact mode) and the LED-mode glitches:
Screenshot_20210814_170819

**Review by BenBE — APPROVED** [review]

**Pull request merged**